swift/src/Ice/LocalExceptions.swift

```diff
@@ -6,14 +6,16 @@
 // Dispatch exceptions
 //
 
-/// The dispatch failed. This is the base class for local exceptions that can be marshaled and transmitted "over the
-/// wire".
+/// The exception that is thrown when a dispatch failed. This is the base class for local exceptions that can be
+/// marshaled and transmitted "over the wire".
+/// You can throw this exception in the implementation of an operation, or in a middleware.
+/// The Ice runtime then logically rethrows this exception to the client.
 public class DispatchException: LocalException, @unchecked Sendable {
     public let replyStatus: UInt8
 
     /// Creates a DispatchException.
     /// - Parameters:
-    ///   - replyStatus: The reply status raw value. It may not correspond to a valid `ReplyStatus` enum value.
+    ///   - replyStatus: The reply status. It may not correspond to a valid ``ReplyStatus`` enum value.
```

Comment:
"raw value" is the correct Swift terminology. It's the raw value of the enumerator.

Reply:
It wasn’t about correctness, I just thought it sounded weird here and was already clear from the type.

I added it back though.